ceremonies/2023/e5-cross-cert.yaml

```diff
@@ -0,0 +1,27 @@
+ceremony-type: intermediate
```

Comment:
I'm realizing just now for the first time that
a) the ceremony tool has a `ceremony-type: cross-certificate` that we could use here; but
b) it's [literally identical to](https://github.com/letsencrypt/boulder/blob/cf770dfdef141c94507200ca803576d530481504/cmd/ceremony/main.go#L794-L803) `ceremony-type: intermediate`.

I think that we should make two improvements here:
1) Change the `ceremony-type: cross-certificate` to behave _almost_ exactly like `ceremony-type: intermediate`, but to additionally take a path to the *existing* certificate as an input. Then it can compare all of its configured values (common name, pubkey, etc) to the input existing cert, to confirm that the cross-certificate being created matches it in every way (except for Issuer and Signature).
2) Change these cross-cert config files to use the new `ceremony-type: cross-certificate` once it exists

Reply:
Fixed by https://github.com/letsencrypt/boulder/pull/7005